### DISCUSSION TITLE
Sidekick20 via Sidekick64

### DISCUSSION THREAD

**Discussion by mwedmark** [frntc/Sidekick64#56] [General]
Would it be possible to run a Sidekick64 in a VIC-20 if you recofigured it with the correct fiwmware and used something like this:
-snip-
Now, as a matter of fact Jim Brain is developing a cartridge device he calls Masquerade, modulo spelling. It will allow you to physically insert a C64 cartridge into a VIC-20, through remapping the I/O lines in the cartridge port.
-snip-

**Comment by frntc** [comment]
I don't remember the details of Masquerade, but essentially it (re)maps some enable-/OE-signals so that simple cartridges work.

The Sidekick20-adapter is a similar thing, but has some special remapping (and combining) of signals -- that is, the two are not the same. I'd have to look into my code if the menu might pop up when using Masquerade, but I don't think so -- and certainly not everything will work.

(but in any case, the SK20-adapter is quickly built, no? :-))
